common/models/user.js

```diff
@@ -558,6 +557,13 @@ module.exports = function(User) {
       return cb.promise;
     }
 
+    try {
+      if (options.password) {
+        UserModel.validatePassword(options.password);
```

Comment:
I have to use `options.password` because otherwise `length` will not be defined and will break few other test cases and cause errors. I mean `length` in main `validatePassword` function.